### ISSUE TITLE
Add DEVELOPMENT.md or CONTRIBUTING.md to document local dev setup

### ISSUE THREAD

**Issue by sarayourfriend** [playwright-community/eslint-plugin-playwright#162]
There are some nuances to contributions and local development that should be documented, including the following:

- How to run tests, including the requirement to run `tsc` before running tests. 
- The requirement to document additions and changes to the rule set.
- The project's usage of use `pnpm`

**Comment by mskelton** [comment]
> How to run tests, including the requirement to run tsc before running tests. Document the strategy of using tsc and jest watch to ease/streamline rule and test development.

This is not true, you don't need to run tsc before tests.

> The project's usage of use pnpm and a request not to commit package-lock.json, yarn.lock etc.

Not something that really needs to be documented explicitly. In the contributing document just use `pnpm` in all the shell command examples and that will make it clear.

**Comment by sarayourfriend** [comment]
I've updated the issue description.

Regarding pnpm and:

> Not something that really needs to be documented explicitly

In my experience with the WordPress/openverse repository and other FOSS projects, it's quite common for inexperienced contributors to either ignore `pnpm-lock.yaml` and default to `npm install` unless it's documented (sometimes still, when it is documented) and they'll frequently commit things like package-lock.json or yarn.lock. In any case, I removed it from the issue description.

I'm a relatively experienced developer and have contributed to various projects and always appreciate some sort of contribution or local dev setup documentation. It reduces the friction to contribution and makes projects more inclusive. Just my 2¢, but feel free to close the issue if you disagree, of course.
